### ISSUE TITLE
how to draw figure 2 in the paper?

### ISSUE THREAD

**Issue by fyting** [czczup/ViT-Adapter#120]
Hi, thx for your great work. Can you share the method to draw the figure 2 in your paper?

**Comment by fyting** [comment]
Thank you for your response. “The table in the lower right corner was added later in latex.”, can you share the latex code?

**Comment by czczup** [comment]
\begin{wrapfigure}{r}{0.5\textwidth}
    \includegraphics[width=0.99\linewidth]{figure/param_ap.pdf}
    
    \vspace{-39.0mm}\hspace{30.2mm}
    \resizebox{0.26\columnwidth}{!}{\tablestyle{2pt}{1}
    	\renewcommand\arraystretch{0.55}
    	\input{table/fig2.tex}
	}
    \vspace{9mm}
    \caption{
    \textbf{Object detection performance on COCO val2017 using Mask R-CNN.} 
    We see that the proposed ViT-Adapter brings significant improvements to plain ViTs.
    $^\bigstar$~indecates using multi-modal pre-trained ViT from~\citep{zhu2021uni}.
    Backbones pre-trained on ImageNet-22K are marked with $^\dagger$, otherwise ImageNet-1K.
    }
    \vspace{-3mm}
    \label{fig:param_ap}
\end{wrapfigure}

**Comment by czczup** [comment]
```
\begin{tabular}[b]{l|c|c}
	% \whline
	\renewcommand{\arraystretch}{0.1}
	Method & \#Param & AP$\rm ^b$ \\
	\hline
	PVTv2-B1   & 33.7M  & 44.9 \\
	ViT-T      & 26.1M  & 40.2 \\
	\rowcolor{gray!20}
	ViT-Adapter-T (ours)            & 28.1M  & 46.0 \\
	\hline
	PVTv2-B2   & 45.0M  & 47.8 \\
	Swin-T     & 47.8M  & 46.0 \\
	ViT-S & 43.8M  & 44.0 \\
	\rowcolor{gray!20}
	ViT-Adapter-S (ours)            & 47.8M  & 48.2 \\
	\hline
	Swin-B     & 107.1M & 48.6 \\
	ViT-B      & 113.6M & 45.8 \\
	\rowcolor{gray!20}
	ViT-Adapter-B (ours)            & 120.2M & 49.6 \\
	\rowcolor{yellow!15}
	ViT-Adapter-B$^\bigstar$(ours)   & 120.2M & 51.2 \\
	\hline
	ViT-L$^\dagger$      & 337.3M & 48.8 \\
	\rowcolor{gray!20}
	ViT-Adapter-L$^\dagger$ (ours)            & 347.9M & 52.1 \\
\end{tabular}
```

**Comment by fyting** [comment]
Thank you very much!
